muuri.js

```diff
@@ -761,7 +761,7 @@
       return [];
     }
 
-    var targetElements = [].concat(elements);
+    var targetElements = Array.isArray(elements) || isNodeList(elements) ? elements : [elements];
```

Comment:
The only problem here is that `indexOf()` method will be called later on in the function context for `targetElements` if there are already elements in the grid. So if `targetElements` is a NodeList it will throw an error. To overcome this we need to convert node list into an array.

Here's my suggestion:
```
var targetElements = isNodeList(elements) ? nodeListToArray(elements) : [].concat(elements);
```